spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -38,6 +57,105 @@
+          FileUtils.cp("#{wixtestfiles}/file-1.wxs", "#{VANAGON_ROOT}/resources/windows/wix/file-1.wxs")
+          cur_plat._platform.generate_msi_packaging_artifacts(workdir,plat[:projname], binding)
+          # check the result
+          expect(File).to exist("#{workdir}/wix/file-1.wxs")
+        end
+
+        it "Picks Project Specific Wix File in favour of Generic Wix file" do
+          # setup source directories and run artifact generation
+          FileUtils.cp("#{wixtestfiles}/file-1.wxs", "#{PROJ_ROOT}/resources/windows/wix/file-wix.wxs")
+          FileUtils.cp("#{wixtestfiles}/file-2.wxs", "#{VANAGON_ROOT}/resources/windows/wix/file-wix.wxs")
+          cur_plat._platform.generate_msi_packaging_artifacts(workdir,plat[:projname], binding)
+          # check the result
+          expect(FileUtils.compare_file("#{wixtestfiles}/file-1.wxs", "#{workdir}/wix/file-wix.wxs")).to be_truthy
+        end
+
+        it "Picks Project Specific Wix File in favour of Generic ERB file" do
+          # setup source directories and run artifact generation
+          FileUtils.cp("#{wixtestfiles}/file-1.wxs", "#{PROJ_ROOT}/resources/windows/wix/file-wix.wxs")
+          FileUtils.cp("#{wixtestfiles}/file-3.wxs.erb", "#{VANAGON_ROOT}/resources/windows/wix/file-wix.wxs.erb")
+          cur_plat._platform.generate_msi_packaging_artifacts(workdir,plat[:projname], binding)
+          # check the result
+          expect(FileUtils.compare_file("#{wixtestfiles}/file-1.wxs", "#{workdir}/wix/file-wix.wxs")).to be_truthy
+        end
+
+        it "Picks Project Specific ERB File in favour of Generic Wix file" do
+          # setup source directories and run artifact generation
+          FileUtils.cp("#{wixtestfiles}/file-3.wxs.erb", "#{PROJ_ROOT}/resources/windows/wix/file-wix.wxs.erb")
+          FileUtils.cp("#{wixtestfiles}/file-2.wxs", "#{VANAGON_ROOT}/resources/windows/wix/file-wix.wxs")
+          cur_plat._platform.generate_msi_packaging_artifacts(workdir,plat[:projname], binding)
+          # check the result
+          expect(FileUtils.compare_file("#{wixtestfiles}/file-3.wxs.erb", "#{workdir}/wix/file-wix.wxs")).to be_truthy
+        end
+
+        it "Picks Project Specific ERB File in favour of Generic ERB file" do
+          # setup source directories and run artifact generation
+          FileUtils.cp("#{wixtestfiles}/file-3.wxs.erb", "#{PROJ_ROOT}/resources/windows/wix/file-wix.wxs.erb")
+          FileUtils.cp("#{wixtestfiles}/file-4.wxs.erb", "#{VANAGON_ROOT}/resources/windows/wix/file-wix.wxs.erb")
+          cur_plat._platform.generate_msi_packaging_artifacts(workdir,plat[:projname], binding)
+          # check the result
+          expect(FileUtils.compare_file("#{wixtestfiles}/file-3.wxs.erb", "#{workdir}/wix/file-wix.wxs")).to be_truthy
+        end
+
+        it "Copies Hierarchy of files from Product Specific Directory to output directory with ERB translation as necessary" do
```

Comment:
The title of the test implies there should be a check for ERB translation, but there is not test.  Is the title of the test wrong or this there an expect missing?